Cargo.toml

```diff
@@ -52,7 +52,8 @@ bitvec = "1"
 blake2s_simd = "1"
 bls12_381 = "0.8"
 jubjub = "0.10"
-sapling = { package = "sapling-crypto", version = "0.1.2" }
+#sapling = { package = "sapling-crypto", version = "0.1.2" }
+sapling = { package = "sapling-crypto", git = "https://github.com/zcash/sapling-crypto", rev = "54fc7d3d9c8eeb8e6ef6f64619deed43b0b681a6" }
```

Comment:
Changed to the sapling version with the derives we need until a release is done there.

Reply:
We can use v0.1.3 here now. https://github.com/zcash/librustzcash/pull/1298/commits/a3b4de0459e2ee443385b7a062910644db6ac400